neuraxle/metaopt/trial.py

```diff
@@ -78,16 +78,22 @@ def __init__(
         self.start_time: datetime.datetime = start_time
         self.end_time: datetime.datetime = end_time
 
-    def new_validation_split(self, pipeline: BaseStep) -> 'TrialSplit':
+    def new_validation_split(self, pipeline: BaseStep, should_delete_pipeline_on_completion: bool = True) -> 'TrialSplit':
```

Comment:
Direct language is more clear and shorter here. 
```suggestion
    def new_validation_split(self, pipeline: BaseStep, delete_pipeline_on_completion: bool = True) -> 'TrialSplit':
```